Branch-SDK/src/main/java/io/branch/referral/network/BranchRemoteInterfaceUrlConnection.java

```diff
@@ -51,6 +49,7 @@ public BranchResponse doRestfulPost(String url, JSONObject payload) throws Branc
     ///-------------- private methods to implement RESTful GET / POST using HttpURLConnection ---------------//
     private BranchResponse doRestfulGet(String url, int retryNumber) throws BranchRemoteException {
         HttpsURLConnection connection = null;
+        String requestId = null;
```

Comment:
Same as below: Variable declaration can be pulled into the try block (where value is assigned) for brevity.

Reply:
I remember this now, this is being used outside of try-catch also, that's why put it here.

Reply:
My bad didn't notice it